chroma/api/local.py

```diff
@@ -69,7 +69,7 @@ def reset(self):
         self._db.reset()
         return True
 
-    def get_nearest_neighbors(self, embedding, n_results, where={}):
+    def get_nearest_neighbors(self, embedding, n_results=10, where={}):
```

Comment:
This is a snuck-in clean up change to match the fastapi n_results. Can remove it not desired.